dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -10,6 +10,7 @@
 - hide_source ||= false
 - use_phaser ||= false
 - use_makerlab ||= false
+- app_options[:locale] = js_locale
```

Comment:
Do you have a suggestion for detecting this other than test coverage and code review?